src/app.config.ts

```diff
@@ -24,17 +24,17 @@ export const configure = (app, setupSwaggerModule = true): OpenAPI.Document => {
     // set cors origin policy from .env
     app.enableCors({
         origin:
-      configService.get('API_CORS_ORIGIN') === 'true' ||
-      configService.get('API_CORS_ORIGIN') === 'false'
-          ? configService.get('API_CORS_ORIGIN') === 'true'
-          : configService.get('API_CORS_ORIGIN'),
+      configService.get('CORS_ORIGIN') === 'true' ||
```

Comment:
wouldn't this ternary always return true?

Reply:
Nope, it is actually just temporary but what it does is let you set cors in .env to either true or false, or to a url string. It returns booleans if env is set to 'true' or 'false' but a string otherwise. I think it also defaults to false if not set. Just for development, will revisit this when looking into auth and security.